### PR TITLE
NO-TICKET - QA. Enable alarms on feature builds and change default verifiers

### PR DESCRIPTION
**Pull request by gslevinkov**

### CODE REVIEW

**Review by kabramovp2p — APPROVED** [review]

File: .github/workflows/pull-request-check.yml [hunk 1]
```diff
@@ -107,7 +109,7 @@ jobs:
           amplitude-key-debug: ${{ secrets.AMPLITUDE_KEY_DEBUG }}
           amplitude-key-release: ${{ secrets.AMPLITUDE_KEY_RELEASE }}
           torus-verifier-debug: ${{ secrets.TORUS_VERIFIER_DEBUG }}
-          torus-verifier-feature: ${{ secrets.TORUS_VERIFIER_FEATURE }}
+          torus-sub-verifier-feature: ${{ secrets.TORUS_SUB_VERIFIER_FEATURE }}
```
Comment (1sakov) [inline .github/workflows/pull-request-check.yml:112]: Why did you delete but not add subverifier?

File: .github/workflows/release-build.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ jobs:
           amplitude-key-debug: ${{ secrets.AMPLITUDE_KEY_DEBUG }}
           amplitude-key-release: ${{ secrets.AMPLITUDE_KEY_RELEASE }}
           torus-verifier-debug: ${{ secrets.TORUS_VERIFIER_DEBUG }}
-          torus-verifier-feature: ${{ secrets.TORUS_VERIFIER_FEATURE }}
+          torus-sub-verifier-feature: ${{ secrets.TORUS_SUB_VERIFIER_FEATURE }}
```
Comment (1sakov) [inline .github/workflows/release-build.yml:29]: same

File: .github/workflows/scheduled-build.yml [hunk 3]
```diff
@@ -24,7 +24,7 @@ jobs:
           amplitude-key-debug: ${{ secrets.AMPLITUDE_KEY_DEBUG }}
           amplitude-key-release: ${{ secrets.AMPLITUDE_KEY_RELEASE }}
           torus-verifier-debug: ${{ secrets.TORUS_VERIFIER_DEBUG }}
-          torus-verifier-feature: ${{ secrets.TORUS_VERIFIER_FEATURE }}
+          torus-sub-verifier-feature: ${{ secrets.TORUS_SUB_VERIFIER_FEATURE }}
```
Comment (1sakov) [inline .github/workflows/scheduled-build.yml:27]: same

File: .scripts/torus.gradle [hunk 4]
```diff
@@ -4,9 +4,11 @@ android {
 
         debug {
             resValue "string", "torusVerifier", "${torusProperties.getOrThrow('TORUS_VERIFIER_DEBUG')}"
+            resValue "string", "torusSubVerifier", "${torusProperties.getOrThrow('TORUS_SUB_VERIFIER_FEATURE')}"
```
Comment (1sakov) [inline .scripts/torus.gradle:7]: I am not sure if we need subverifier for debug builds. 
There is different verifier

**Review by 1sakov — CHANGES_REQUESTED** [review]
Do I need to add subverifier into CI secret variables?

File: .scripts/torus.gradle (hunk 4 above)
Comment (gslevinkov) [reply .scripts/torus.gradle:7]: i'll get you an update on the changes with verifiers

**Comment by gslevinkov** [comment]
> Do I need to add subverifier into CI secret variables?

no need for that, i already did it

File: .github/workflows/release-build.yml (hunk 2 above)
Comment (gslevinkov) [reply .github/workflows/release-build.yml:29]: yeah, strange
other files were ok
fixed

**Review by 1sakov — APPROVED** [review]
